.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: run script.py
+
+on:
+  schedule:
+    - cron: '0 */2 * * *' # Every Two Hours 
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: checkout repo content
+        uses: actions/checkout@v2 # checkout the repository content to github runner
```

Comment:
This version is outdated. Please update it to latest, same for all actions. Also add the dependabot.yaml and the PR titles checker action we have in all repositories here (see backend-monorepo fir example)